.templates/setup.cfg.template

```diff
@@ -0,0 +1,26 @@
+{% extends "templates/setup.cfg.template" %}
+
+{% block flake8 %}
+[flake8]
+exclude =
+    __init__.py
+    {% for exclude in flake8.exclude %}
+    {{ exclude }}
+    {% endfor %}
+ignore =
+    E123
+    E133
+    E203
+    E226
+    E241
+    E242
+    E501
+    E731
+    F401
+    W503
+    {% for ignore in flake8.ignore %}
+    {{ ignore }}
+    {% endfor %}
+max-complexity = {{ flake8.max_complexity }}
```

Comment:
Is this just overridden so that the max complexity can be changed? If so, I would probably just use the default template and use `noqa: C901` to disable that check for particular functions instead.

Reply:
Yeah, there's just a lot of them. I was wondering about allowing it to be changed in nengo-bones, for repos that don't care as much about it (just like they can disable particular pylint errors).

Reply:
You can just disable the complexity check as well (setting the max complexity to 30 seems like kind of the same as disabling it :wink:)

Reply:
Sorry I meant just disabling the C901 check globally, didn't mean to make you go through and label all the functions. But you did it already now though so 🥇 .